pyoos/collectors/nerrs/nerrs_soap.py

```diff
@@ -30,6 +45,8 @@ def get_stations(self):
                 if val is None:
                     val = ""
                 s[child.tag] = val
+            s['Longitude'] = -float(s['Longitude'])
```

Comment:
> Is this safe?

All longitudes are being reported as West values, so positive. My commit corrects a mistake in the pyoos nerrs collector, b/c it was assuming they were negative (a reasonable expectation, except it was wrong). NERRS is a network of coastal research reserves in the US (including Alaska and Puerto Rico), so it's safe to say that there are no plans to have sites with East longitudes.